libeos-payg/provider.h

```diff
@@ -35,6 +35,7 @@ struct _EpgProviderInterface
   gboolean        (*add_code)   (EpgProvider  *self,
                                  const gchar  *code_str,
                                  guint64       now_secs,
+                                 gint64       *time_added,
```

Comment:
We'd need a corresponding PR for `eos-payg-nonfree` to add this parameter to that provider.

Reply:
https://github.com/endlessm/eos-payg-nonfree/pull/8